src/DynamoCoreWpf/DynamoCoreWpf.csproj

```diff
@@ -75,7 +75,6 @@
     <Reference Include="PresentationFramework" />
     <Reference Include="ProtoGeometry, Version=2.12.0.5595, Culture=neutral, processorArchitecture=AMD64">
       <HintPath>..\packages\DynamoVisualProgramming.LibG_227_0_0.2.12.0.5595\lib\net48\ProtoGeometry.dll</HintPath>
-      <Private>False</Private>
```

Comment:
@M-JULIANI not sure if it makes any difference, but why is the `<Private>` tag removed from here?

Reply:
I would not remove this tag

Reply:
Done.